crates/language/src/language.rs

```diff
@@ -84,7 +84,7 @@ pub fn init(cx: &mut AppContext) {
 }
 
 thread_local! {
-    static PARSER: RefCell<Parser> = {
+    pub static PARSER: RefCell<Parser> = {
```

Comment:
I wonder if we should make an accessor function for this rather than exposing the `static` like this? Or maybe even expose a higher-level `parse` function?